WeakAuras/GenericTrigger.lua

```diff
@@ -829,9 +829,21 @@ function GenericTrigger.Add(data, region)
             else
               error("Improper arguments to WeakAuras.Add - no event prototype can be found for event type \""..trigger.event.."\" and default prototype reset failed.");
             end
-          elseif(trigger.event == "Combat Log" and not (trigger.subeventPrefix..trigger.subeventSuffix)) then
```

Comment:
This cannot ever happen. If we succuesfully concanated prefix and suffix, then that value can never be nil.